girs/Adw-1.gir

```diff
@@ -5151,6 +5151,9 @@ If the value is 0, the number of lines won't be limited.</doc>
         <type name="PreferencesRowClass" c:type="AdwPreferencesRowClass"/>
```

Comment:
GIRs as a submodule might make sense in the future. `types` we usually have to update in MRs, but `girs` does not have as tight a dependency.

Reply:
Alternatively I think you can flag files as "binary" to avoid them spamming MR review but I'm not sure that is ideal unless we have a CI check that validates the GIRs.

Reply:
yes, it makes sense to move them to a submodule, I can do that

Reply:
maybe the gir files are also interesting for other projects (like we used the ones from vala), so I think a submodule would be useful

Reply:
Would be good as a follow up